docs/terraform.md

```diff
@@ -10,7 +10,7 @@
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 5.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 5.25.0 |
```

Comment:
This is an annoying bug with terraform-docs. You may want to update manually and push as this will fail tests.

Reply:
Thanks @Gowiem! That fixed it.